WIC/Constants/WICPixelFormats.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+
+namespace WIC.Constants
+{
+    public static class WICPixelFormats
```

Comment:
Consider renaming this class to `WICPixelFormat` (without `s`) for consistency with other preexisting enum-like typed in `Constants/`.